lib/fishjam/rpc_client.ex

```diff
@@ -51,4 +65,9 @@ defmodule Fishjam.RPCClient do
   end
 
   defp nodes, do: [Node.self() | Node.list()]
+
+  defp emit_rpc_duration_event(event_name, start_time) do
+    duration = System.monotonic_time(:millisecond) - start_time
+    :telemetry.execute(event_name, %{duration: duration})
```

Comment:
Again, this should be in seconds; you can use `System.monotonic_time()` and perform the conversion automatically by specifying `unit: {:native, :second}` in `telemetry.ex`, just like in the Phoenix metrics